tests/test_shell_bash_background_helpers_disowned.py

```
@@ -0,0 +1,217 @@
+        version = _bash_version(candidate)
+        if version is not None and version >= (5, 3):
+            return candidate
+    return None
+
+
+class BoundUnixSocket:
+    def __init__(self, path: Path) -> None:
+        self.path = path
+        self.sock: socket.socket | None = None
+
+    def __enter__(self) -> "BoundUnixSocket":
```

Ruff (0.15.13)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)